next/pages/nases/login.tsx

```diff
@@ -24,27 +28,45 @@ const NasesLoginPage = () => {
     }
     effectOnceRan.current = true
     const metadata = popSendEidMetadata()
-    if (!metadata) {
+    const metadataVerify = popSendEidMetadataVerify()
+    if (!metadata && !metadataVerify) {
       // eslint-disable-next-line @typescript-eslint/no-floating-promises
       router.push(ROUTES.HOME)
       return
     }
 
     const { token } = router.query
-    const query = {
-      [STEP_QUERY_PARAM_KEY]: STEP_QUERY_PARAM_VALUE_SUMMARY,
-      ...(typeof token === 'string' ? { [FORM_SEND_EID_TOKEN_QUERY_KEY]: token } : {}),
+
+    if (metadataVerify) {
```

Comment:
Keeping the same order, 1. send, 2. verify, please move this if block below `if (metadata)` block. I find it helpful for easier orientation in the code in genera